conf/config-defaults.php

```diff
@@ -87,6 +87,8 @@
 $Configuration['Garden']['Security']['Hsts']['Preload'] = false;
 $Configuration['Garden']['Security']['Hsts']['MaxAge'] = 604800;
 
+$Configuration['Garden']['Security']['SourceMaps']['Enabled'] = false;
```

Comment:
```suggestion
```

I think having nothing in config-defaults would be a better default. That way, unless explicitly configured it will map to the debug value.

With this setup, the fallback value specified in the in the static controller above will never be reached.